ceph/ceph.py

```diff
@@ -2107,6 +2107,15 @@ def download_file(self, src, dst, sudo=False):
         client = self.rssh if sudo else self.ssh
         client().open_sftp().get(src, dst)
 
+    def create_dirs(self, dir_path, sudo=False):
+        """Create directory on node
+        Args:
+            dir_path (str): Directory path to create
+            sudo (bool): Use root access
+        """
+        cmd = "mkdir -p {}".format(dir_path)
```

Comment:
can we use [1] instead 
[1] https://docs.paramiko.org/en/stable/api/sftp.html#paramiko.sftp_client.SFTPClient.mkdir